macros/edr/dbt_artifacts/upload_dbt_invocation.sql

```diff
@@ -41,6 +47,54 @@
     {% do return(config.project_name) %}
 {% endmacro %}
 
+{% macro get_env() %}
+    {% set env = elementary.get_config_var("env") %}
+    {% if env %}
+        {{ return(env) }}
+    {% endif %}
+    {{ return(elementary.get_first_not_none_env_var(["DBT_ENV"])) }}
+{% endmacro %}
+
+{% macro get_job_id() %}
+    {% set job_id = elementary.get_config_var("job_id") %}
```

Comment:
@oravi 
Do you think those should also be just env vars like `JOB_ID`?
Most likely that if they'd add a var it'll just be
```yml
vars:
  job_id: "{{ env_var('JOB_ID') }}"
```